BUILDING.md

```diff
@@ -130,6 +130,12 @@ Depending on the host platform, the selection of toolchains may vary.
 #### Windows
 
 * Visual Studio 2017 with the Windows 10 SDK on a 64-bit host.
+    * Targeting Windows on ARM (ARM64 Win32) requires
+        * Visual Studio 15.9.0+
+        * Visual Studio components
+            * Visual C++ compilers and libraries for ARM64
+            * Visual C++ ATL for ARM64
```

Comment:
This section is about what toolchains are supported. Wouldn't it be better to remove these 3 lines about the exact components, and add an entry below under Windows prerequisites?

Reply:
Yup, this is in the wrong section. I've updated that part of the PR.